osmosis-installer.py

```diff
@@ -582,10 +750,12 @@ def initSetup ():
         os.chdir(os.path.expanduser(HOME+'/osmosis'))
         if networkAns == "1":
             print(bcolors.OKGREEN + "(5/5) Installing Osmosis V7.0.3 Binary..." + bcolors.ENDC)
+            subprocess.run(["git stash"], stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL, shell=True)

```

Comment:
q: How did it work before without git stash? Why do we need it now?

Reply:
Most of the time people ran this script on a fresh VM, and even if they didn't they usually didn't make local changes to the osmosis folder. I only realized after having to directly modify the go.mod to accept rocksdb that you cant automate git checkout if you don't stash first